sql/core/src/test/resources/sql-tests/results/describe-query.sql.out

```diff
@@ -154,16 +154,46 @@ DESCRIBE
 
 
 -- !query 15
-DROP TABLE desc_temp1
+EXPLAIN DESC QUERY SELECT * FROM desc_temp2 WHERE key > 0
 -- !query 15 schema
-struct<>
+struct<plan:string>
 -- !query 15 output
-
+== Physical Plan ==
+Execute DescribeQueryCommand
+   +- DescribeQueryCommand SELECT * FROM desc_temp2 WHERE key > 0
 
 
 -- !query 16
-DROP TABLE desc_temp2
+EXPLAIN EXTENDED DESC WITH s AS (SELECT 'hello' as col1) SELECT * FROM s
 -- !query 16 schema
-struct<>
+struct<plan:string>
 -- !query 16 output
+== Parsed Logical Plan ==
+DescribeQueryCommand WITH s AS (SELECT 'hello' as col1) SELECT * FROM s
+
+== Analyzed Logical Plan ==
+col_name: string, data_type: string, comment: string
+DescribeQueryCommand WITH s AS (SELECT 'hello' as col1) SELECT * FROM s
+
+== Optimized Logical Plan ==
+DescribeQueryCommand WITH s AS (SELECT 'hello' as col1) SELECT * FROM s
+
+== Physical Plan ==
+Execute DescribeQueryCommand
+   +- DescribeQueryCommand WITH s AS (SELECT 'hello' as col1) SELECT * FROM s
```

Comment:
One minor concern about this is that the raw SQL string approach might look confusing for the multi-line queries. For example, I can make the following. The surrounding comments and whitespaces are removed, but the comment and newlines in the statement survive as expected.

```scala
scala> sql("EXPLAIN EXTENDED DESC /* this is gone */ SELECT 1\n /* this survives */ \nFROM /* last */\n VALUES(1) /* this is removed, too */").show(false)

|== Parsed Logical Plan ==
DescribeQueryCommand SELECT 1
 /* this survives */
FROM /* last */
 VALUES(1)

== Analyzed Logical Plan ==
col_name: string, data_type: string, comment: string
DescribeQueryCommand SELECT 1
 /* this survives */
FROM /* last */
 VALUES(1)

== Optimized Logical Plan ==
DescribeQueryCommand SELECT 1
 /* this survives */
FROM /* last */
 VALUES(1)

== Physical Plan ==
Execute DescribeQueryCommand
   +- DescribeQueryCommand SELECT 1
 /* this survives */
FROM /* last */
 VALUES(1)
|
```

Reply:
@dongjoon-hyun Oh.. i hadn't tried with surrounding comments. Thanks for finding it. Would you have any suggestion here ?

Reply:
@dongjoon-hyun I thought about this some more. Actually i think there is an ambiguity whether the "surrounding" comments are part of the explain statement or the query. In order to verify the theory, i tried the following : 
```
create view vtest as /* leading */ select /* embeded */ 1 FROM VALUES(1) /* trailing */;
spark-sql> show table extended  like 'vtest';
default	vtest	false	Database: default
Table: vtest
Owner: dbiswal
Created Time: Fri Apr 19 14:36:32 PDT 2019
Last Access: Wed Dec 31 16:00:00 PST 1969
Created By: Spark 3.0.0-SNAPSHOT
Type: VIEW
View Text: select /* embeded */ 1 FROM VALUES(1)
View Original Text: select /* embeded */ 1 FROM VALUES(1)
View Default Database: default
View Query Output Columns: [1]
Table Properties: [transient_lastDdlTime=1555709792, view.query.out.col.0=1, view.query.out.numCols=1, view.default.database=default]
Serde Library: org.apache.hadoop.hive.serde2.lazy.LazySimpleSerDe
InputFormat: org.apache.hadoop.mapred.SequenceFileInputFormat
OutputFormat: org.apache.hadoop.hive.ql.io.HiveSequenceFileOutputFormat
Storage Properties: [serialization.format=1]
Schema: root
 |-- 1: integer (nullable = false)
```
As we see here the original query does not contain the surrounding comments. Given this, i think we would be okay in dropping the surrounding comments and preserving the embedded comments. What do you think ?

Reply:
+1 for the reasoning, @dilipbiswal . Thanks.